packages/react-microlink/stories/index.js

```diff
@@ -139,6 +139,18 @@ storiesOf('Normal', module)
       }}
     />
   ))
+  .addWithJSX('with reverse prop', () => (
+    <div>
+      {urls.map(url =>
+        <MicrolinkCard
+          key={url}
+          url={url}
+          style={{marginBottom: '20px'}}
+          reverse
+        />
+      )}
+    </div>
+  ))
   .addWithJSX('with empty state', () => (
```

Comment:
I would like keep `empty` as the final story 🙂

Reply:
Yeah - me too 😛, I've left it like that, right?